server/repository/src/migrations/v1_01_01/mod.rs

```diff
@@ -32,6 +33,42 @@ impl Migration for V1_01_01 {
     }
 }
 
+#[cfg(test)]
+async fn setup_data_migration(name: &str) -> StorageConnection {
```

Comment:
this was used in `remove check from sqlite` work, but also would be needed in tests for splitting inventory adjustment migration, so relocated here